resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -70,6 +70,13 @@ package object config {
     .booleanConf
     .createWithDefault(false)
 
+  private[spark] val POPULATE_HADOOP_CLASSPATH = ConfigBuilder("spark.yarn.populateHadoopClasspath")
+    .doc("Whether to populate Hadoop classpath from `yarn.application.classpath` and " +
```

Comment:
we might add a comment saying requires bundling Hadoop with spark or user to provide Hadoop install separately.
Also need to add this to configuration.md doc

Reply:
+1 for the comment and the doc.